parser/model/ddl.go

```diff
@@ -747,6 +747,12 @@ func (job *Job) IsPaused() bool {
 	return job.State == JobStatePaused
 }
 
+// IsPausedBySystem returns whether the job is paused by system.
+func (job *Job) IsPausedBySystem() bool {
+	// TODO: Replace JobStatePausing with JobStatePaused after merging https://github.com/pingcap/tidb/pull/43297..
+	return job.State == JobStatePausing && job.AdminOperator == AdminCommandBySystem
```

Comment:
IsPaused() || IsPausing() would be better.

Reply:
paused, it looks like it's okay for now, because I'm just using this one. The other thing is the function name is a little bit simpler, right?